hal/Cargo.toml

```diff
@@ -109,6 +114,8 @@ samd51j19a = ["atsamd51j19a", "samd51"]
 samd51j19a-rt = ["atsamd51j19a", "atsamd51j19a/rt", "samd51"]
 samd51j20a = ["atsamd51j20a", "samd51"]
 samd51j20a-rt = ["atsamd51j20a", "atsamd51j20a/rt", "samd51"]
+samd51n20a = ["atsamd51n20a", "samd51"]
```

Comment:
As a general question, would it make sense to add a set of features for `samd51g`, `samd51j`, `samd51n` and `samd51p` to cover the different pinouts?

Reply:
I think that might be a good idea, I was actually thinking the same the other day. @sajattack @jacobrosenthal thoughts?